### PR TITLE
Meta: fix auto-deployment

### PR DESCRIPTION
**Pull request by bakkot**
The [current deploy script](https://github.com/tc39/ecma262/blob/f4694412639e19acab2c0dd5baac247aa7d7a889/scripts/auto-deploy.sh#L5) relies on secrets in Travis to decrypt the deploy key, which no longer works now that we are not using Travis.

Instead, just use the built-in token.

### CODE REVIEW

**Review by michaelficarra — APPROVED** [review]

**Comment by devsnek** [comment]
could we use github-pages-deploy-action here?

**Comment by bakkot** [comment]
> could we use github-pages-deploy-action here?

Is there a reason to do so?

**Comment by ljharb** [comment]
Thanks, nice catch

File: scripts/auto-deploy.sh [hunk 1]
```diff
@@ -1,36 +1,21 @@
...
-# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
-# Enable SSH authentication
-
-chmod 600 ~/.ssh/$PRIVATE_KEY_FILE_NAME
-echo "Host github.com" >> ~/.ssh/config
-echo "  IdentityFile ~/.ssh/$PRIVATE_KEY_FILE_NAME" >> ~/.ssh/config
-
-# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+declare -r PRIVATE_KEY_FILE_NAME='github_deploy_key'
```
Comment (bakkot) [inline scripts/auto-deploy.sh:11]: Oh, this is stray, let me kill it

**Comment by devsnek** [comment]
@bakkot just to reduce bespoke shell script stuff, not needed of course.

File: scripts/auto-deploy.sh (hunk 1 above)
Comment (bakkot) [reply scripts/auto-deploy.sh]: OK done, sorry for churn

**Comment by bakkot** [comment]
@devsnek Ah, sure. I think our particular use case is simple enough that it doesn't warrant a new dependency.

**Pull request merged**